sources/packages/api/src/route-controllers/restriction/restriction.aest.controller.ts

```diff
@@ -205,15 +209,19 @@ export class RestrictionController extends BaseController {
    * @param studentRestrictionId
    * @param payload
    */
-  @Groups(UserGroups.AESTUser)
-  @AllowAuthorizedParty(AuthorizedParties.aest)
   @Roles(Role.StudentResolveRestriction)
+  @ApiUnprocessableEntityResponse({
+    description:
+      "Resolution Notes are mandatory to resolve the restriction." +
+      "The restriction neither assigned to student nor active. Only active restrictions can be resolved." +
+      "The given restriction type is not Provincial. Only provincial restrictions can be resolved by application user.",
+  })
   @Patch("/student/:studentId/studentRestriction/:studentRestrictionId/resolve")
   async resolveStudentProvincialRestriction(
```

Comment:
Can you please add a todo to have this method refactored to save both operations on a transaction.
![image](https://user-images.githubusercontent.com/61259237/184969378-b2a4a46a-aee3-4283-81ae-82d9b35a9ca3.png)